tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -107,4 +127,66 @@ var _ = Describe("odo devfile describe command tests", func() {
 		})
 	})
 
+	Context("devfile has missing metadata", func() {
+		// Reference: https://github.com/openshift/odo/issues/4815
+		var metadata devfilepkg.DevfileMetadata
+
+		// copyAndCreate copies required source code and devfile, and creates a component
+		var copyAndCreate = func(path string) {
```

Comment:
But it takes only one argument `path`. How does it figure out both source code and devfile using it? 🤔

Reply:
Both the devfiles in use are of java-springboot, so the source code remains the same. `path` is just the file name. I guess I should change the variable name in this case to make more sense.